src/overlays/actors/ovl_En_Arrow/z_en_arrow.c

```diff
@@ -250,7 +250,6 @@ void EnArrow_Fly(EnArrow* this, GlobalContext* globalCtx) {
     Actor* hitActor;
     Vec3f sp60;
     Vec3f sp54;
-
```

Comment:
Were all these line breaks removed on purpose? I think they improve readability, so we should probably keep them